latest/root/usr/bin/run-postgresql-slave

```diff
@@ -28,9 +28,13 @@ check_env_vars
 generate_passwd_file
 generate_postgresql_config
 
+process_extending_files ${APP_DATA}/src/pre-init ${CONTAINER_SCRIPTS_PATH}/pre-init
+
 wait_for_postgresql_master
 export MASTER_FQDN=$(postgresql_master_addr)
 initialize_replica
 
+process_extending_files ${APP_DATA}/src/init ${CONTAINER_SCRIPTS_PATH}/init
```

Comment:
Should slave call these hooks?

Reply:
I asked for it to be added because I thought even slaves want to have their configuration files extendible. But looking at pg_basebackup documentation it seems that the slaves will get the master's configurations files along with everything else, so it might not be needed after all.

Reply:
More it seems to me that the password change method is newly called for slave, too.